docs/deployment/README.md

```diff
@@ -333,6 +333,20 @@ If you've lost track of the needed redirect URIs, they can be located on the [AP
 ----
 ## Deploy Cumulus dashboard
 
+### Dashboard Requirements
+
+Please note that the requirements are similar to the [Cumulus stack deployment requirements](#Requirements), however the node version may vary slightly and the dashboard requires yarn.    The installation instructions below include a step that will install/use the required node version referenced in the `.nvmrc` file in the dashboard repository.
```

Comment:
This link isn't taking me up to the requirements.

Reply:
46df58a